gradle.properties

```diff
@@ -3,7 +3,7 @@ version=0.4.4
 
 kotlin_version=1.7.20-Beta
 jmhVersion=1.21
-infra_version=0.3.0-dev-64
+infra_version=0.3.0-dev-69
```

Comment:
This version doesn't look like released from master branch

Reply:
I have reverted this change.